wagtail/admin/panels/field_panel.py

```diff
@@ -129,19 +184,29 @@ def is_shown(self):
             return True
 
         def is_required(self):
+            if self.bound_field is None:
+                return False
             return self.bound_field.field.required
 
         def classes(self):
-            is_streamfield = isinstance(self.bound_field.field, BlockField)
-            extra_classes = ["w-panel--nested"] if is_streamfield else []
-
-            return self.panel.classes() + extra_classes
+            classes = self.panel.classes()
+            if self.bound_field and isinstance(self.bound_field.field, BlockField):
+                classes.append("w-panel--nested")
+            return classes
 
         @property
         def icon(self):
             """
             Display a different icon depending on the field's type.
             """
+            if self.panel.icon:
```

Comment:
Have these changes been introduced by mistake in a rebase? They seem to be unrelated to the rest of the PR, and partially match the changes made in #10320.

Reply:
I'm pretty sure I was just rewriting to better handle cases where `self.bound_field` is `None` (which is always the case for read-only panels)